tools/src/bin/output-file.rs

```diff
@@ -266,6 +266,14 @@ fn main() {
                 accel_key: None,
             });
         }
+        if let Some(ref ccov_root) = tree_config.paths.ccov_root {
+            tools_items.push(PanelItem {
+                title: "Code Coverage".to_owned(),
+                link: format!("{}#revision=latest&path={}", ccov_root, path),
```

Comment:
Should this be revision-aware rather than assuming `latest`?

Reply:
We could do that, but right now the Mercurial revision is not available here (we only have the git one).
We could use the mapper, but I think we can do that in a follow-up (adding support for HG Web and DXR too, which are both pointing to "tip").

Reply:
Sure, sounds good.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1585266.